collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/ExternalSecondaryInstanceTest.java

```diff
@@ -43,4 +42,15 @@ public void displaysAllOptionsFromSecondaryInstance() {
                 .assertText("ab")
                 .assertText("ac");
     }
+
+    @Test
+    public void displaysAllOptionsFromCSVSecondaryInstance() {
```

Comment:
As I understand the only difference between this test and the one above is that one uses external XML file and another csv file? If so please correct the name of the test above so that it's clear it's about a secondary XML file.
It would be also easier to understand if both files contained the same data.